### PR TITLE
applicant and shortlist count

### PR DESCRIPTION
**Pull request by amalesic**
added the applicant and shortlist count to the homepage

### CODE REVIEW

File: submitter/templates/submitter/homepage.html
Comment (SunilCotterill) [inline submitter/templates/submitter/homepage.html:1]: &nbsp; doesn't scale

Comment (SunilCotterill) [reply submitter/templates/submitter/homepage.html:1]: Lol github doesn't recognize & n b s p ;

Comment (SunilCotterill) [reply submitter/templates/submitter/homepage.html:1]: But that's just adding manufactured whitespace

**Comment by juliafogerty** [comment]
the nbsp was my b when i was a wee learner

File: submitter/templates/submitter/homepage.html
Comment (juliafogerty) [reply submitter/templates/submitter/homepage.html:1]: the nbsp was my b when i was a wee learner

**Review by juliafogerty — APPROVED** [review]
Added back tab space, sunil said this would not scale but it seemed fine when i tested it. I think it should be good.

**Pull request merged**